muzlag/pkg/config/process.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+var ErrProcessForNotPointer = errors.New("process not possible for non-pointers")
+
+func Process(appName string, cfg any) error {
```

Comment:
Why do you need it?

Reply:
what do you mean? 
Proccess for parse envconfig
appName because in future it can be reuse fo another bot

Reply:
But this function only wraps enconfig.Process, and doesn't really do any useful work

Reply:
ok, I'll delete it